docs/UserGuide.md

```diff
@@ -279,6 +279,11 @@ Format: `mark INDEX`
 * The index refers to the index number shown in the currently displayed company list.
 * The index **must be a positive integer** 1, 2, 3, …​
 
+  Examples:
+* `list` followed by `mark 2` marks the 2nd company in the intern book.
+* `find Google` followed by `mark 1` marks the 1st company in the results of the `find` command.
```

Comment:
i think the could be consistent with the unmark, at the end could also include, "if it is unmarked"